src/app.rs

```diff
@@ -144,195 +147,204 @@ impl App {
-                    .long_help("Specify when to use the decorations that have been specified \
+        ).arg(
+            Arg::with_name("decorations")
+                .long("decorations")
+                .overrides_with("decorations")
+                .takes_value(true)
+                .value_name("when")
+                .possible_values(&["auto", "never", "always"])
+                .default_value("auto")
+                .help("When to show the decorations specified by '--style'.")
+                .long_help("Specify when to use the decorations that have been specified \
                                 via '--style'. The automatic mode only enables decorations if \
                                 an interactive terminal is detected."),
-            ).arg(
-                Arg::with_name("paging")
-                    .long("paging")
-                    .overrides_with("paging")
-                    .takes_value(true)
-                    .value_name("when")
-                    .possible_values(&["auto", "never", "always"])
-                    .default_value("auto")
-                    .help("Specify when to use the pager.")
-                    .long_help("Specify when to use the pager. To control which pager \
+        ).arg(
+            Arg::with_name("paging")
+                .long("paging")
+                .overrides_with("paging")
+                .takes_value(true)
+                .value_name("when")
+                .possible_values(&["auto", "never", "always"])
+                .default_value("auto")
+                .help("Specify when to use the pager.")
+                .long_help("Specify when to use the pager. To control which pager \
                                 is used, set the PAGER or BAT_PAGER environment \
                                 variables (the latter takes precedence). The default \
                                 pager is 'less'. To disable the pager permanently, set \
                                 BAT_PAGER to an empty string."),
+        ).arg(
+            Arg::with_name("wrap")
+                .long("wrap")
+                .overrides_with("wrap")
+                .takes_value(true)
+                .value_name("mode")
+                .possible_values(&["auto", "never", "character"])
+                .default_value("auto")
+                .help("Specify the text-wrapping mode.")
+                .long_help("Specify the text-wrapping mode."),
+        ).arg(
+            Arg::with_name("unbuffered")
+                .short("u")
+                .hidden_short_help(true)
+                .long_help(
+                    "This option exists for POSIX-compliance reasons ('u' is for \
+                         'unbuffered'). The output is always unbuffered - this option \
+                         is simply ignored.",
+                ),
+        ).arg(
+            Arg::with_name("tabs")
+                .long("tabs")
+                .short("t")
```

Comment:
I would like to reserve the short `-t` for now. Maybe we want to have full compatibility with GNU cat at some point (which uses `-t` for something else).

Reply:
How about `-x` for expand, since `-e` is already used? Or should I remove the short option entirely?

Reply:
I think we can remove the short version. It's not something that people will likely modify all the time.

Reply:
Will do.